tools/pkgchk/packages/verifiers.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package packages
```

Comment:
Please remove this file - this has been refactored to `tools/internal/packages`

Reply:
fixed